qiita_pet/templates/study_ajax/sample_summary.html

```diff
@@ -0,0 +1,78 @@
+{% from future.utils import viewitems %}
```

Comment:
I think the idea was to eventually move to py3 only (we are currently not even py3 compatible). In interest of time, fine with leaving as it is.